src/importexport/midi/internal/midishared/midifile.cpp

```diff
@@ -587,9 +586,8 @@ bool MidiFile::readEvent(MidiEvent* event)
             LOGD("readEvent: error 6");
```

Comment:
Shouldn't/Couldn't the abobe `if` be an `else if`?
```cpp
    if (me == 0xf0 || me == 0xf7) {
...
    } else if (me == ME_META) {
...

Reply:
@Jojo-Schmitz I think you are right and that should not introduce any issues.

```
if (me == 0xf0 || me == 0xf7) {
    // SysEx handling
    ...
    return ..
} else if (me == ME_META) {
    // Meta handling
    ...
    return ...
}
if (me & 0x80) { 
    ... 
}
```

Reply:
Probably this small optimization can go into a separate PR

Reply:
Don't see why, but maybe into a separate cvommit of this PR